scene/main/http_request.cpp

```diff
@@ -266,6 +276,16 @@ bool HTTPRequest::_handle_response(bool *ret_value) {
 				final_body_size.set(0);
 				redirections = new_redirs;
 				*ret_value = false;
+				if (method != HTTPClient::METHOD_GET && method != HTTPClient::METHOD_HEAD && method != HTTPClient::METHOD_OPTIONS && method != HTTPClient::METHOD_TRACE) {
```

Comment:
Again, this conditional is quite large due to having to namespace all of the constants. Can you break this out in a separate function like, `_is_method_safe` or something?